api/cds.proto

```diff
@@ -17,6 +17,10 @@ service ClusterDiscoveryService {
       returns (stream ClusterDiscoveryResponse) {
   }
 
+  rpc BidiStreamClusters(stream ClusterDiscoveryRequest)
+      returns (stream ClusterDiscoveryResponse) {
```

Comment:
I would vote to just replace the server stream-only version `StreamClusters` with this implementation and drop the `Bidi` from the name. Same with other xDS. This is one less stub to implement and this will also simplify client implementation.